src/UtilityTrait.php

```diff
@@ -403,19 +409,21 @@ private function getOption($option)
     private function getBaseUrl()
     {
         $version = $this->getOption('version');
+        $domain = $this->getOption('domain');
+        $scheme = 'https://';
```

Comment:
I think it would make more sense to call it `protocol`

Reply:
Urls contain scheme which are associated with protocols. I believe`https` here indicates that request should be served using HTTP protocol. For instance, `file` would indicate that request should be served using FTP protocol.

Or at least that is my understanding and I might be wrong :D

Reply:
I suppose you're right, while HTTPS is a protocol https:// cannot really be called a protocol since it only specifies which protocol to use.